### ISSUE TITLE
not an issue but

### ISSUE THREAD

**Issue by zvchiee** [photopea/photopea#2489]
can you please make it so photopea works on ios 14? i dont have anything else to edit on

**Comment by photopea** [comment]
Yes, it is in our todo list here: https://github.com/photopea/photopea/issues/2447

I am at the hospital now and it would take a week or two to get out. Then, I am planning to borrow an iPhone from my friend and look closer at this problem.

The worst thing is, that Safari is the only browser for iOS, and when it stops working, iOS users have no solution.
